src/raftstore/store/peer.rs

```diff
@@ -379,6 +389,20 @@ impl Peer {
         Ok(())
     }
 
+    fn transfer_leader(&mut self, cmd: RaftCmdRequest) -> Result<()> {
```

Comment:
`transfer_leader` and `exec_transfer_leader` confuses me, why not merge two methods?